packages/core/src/components/discord-embed/DiscordEmbed.ts

```diff
@@ -328,10 +333,15 @@ export class DiscordEmbed extends LitElement implements DiscordEmbedProps, Light
 									${when(
 										this.authorUrl,
 										() =>
-											html`<a href=${ifDefined(this.authorUrl)} target="_blank" rel="noopener noreferrer">
-												${emojiParsedAuthorName}
+											html`<a
+												href=${ifDefined(this.authorUrl)}
+												target="_blank"
+												rel="noopener noreferrer"
+												class="discord-embed-author-block"
+											>
+												<span class="discord-embed-author-block">${emojiParsedAuthorName}</span>
 											</a>`,
-										() => html`${emojiParsedAuthorName}`
+										() => html`<span class="discord-embed-author-block">${emojiParsedAuthorName}</span>`
```

Comment:
You added inline styles here. That makes it harder for end-user customizability and we try to avoid them unless strictly necessary. I moved the styles to a class.